tests/fixtures/versions.py

```diff
@@ -0,0 +1,12 @@
+import json
+import pathlib
+
+
+def versions_data():
+    """
+    Returns a dictionary representing the dictionary
+    we'd expect returned from store.get_datasets().
```

Comment:
well done on this, you just need to update the docsting to store.get_versions().